### PR TITLE
fix(fastai): use TrackerCallback default monitor value

### LINKED ISSUE #442: fastai: wandb not logging fastai validation loss with default args
`wandb --version && python --version && uname`

* Weights and Biases version: 0.8.5
* Python version: 3.7.3
* Operating System: Linux
* fast.ai version: 1.0.55

### Description

When you use the `fastai` callback `WandbCallback` with the default arguments you get some errors and the validation loss is not logged.

### What I Did

1. Run the following code in a jupyter notebook
```
import wandb
import fastai
from wandb.fastai import WandbCallback
from fastai.vision import *
from functools import partial
print(f'wandb version: {wandb.__version__}, fastai version: {fastai.__version__}')
wandb.init(project="fastai-test")
path = untar_data(URLs.MNIST_SAMPLE)
data = ImageDataBunch.from_folder(path)
learn = cnn_learner(data, models.resnet18, metrics=accuracy, callback_fns=WandbCallback)
learn.fit_one_cycle(1, 1e-2)
```

2. You'll get the following error:

image

```
/home/fastai/anaconda3/envs/wandbtest/lib/python3.7/site-packages/fastai/callbacks/tracker.py:50: UserWarning: <class 'wandb.fastai.WandbCallback'> conditioned on metric `val_loss` which is not available. Available metrics are: train_loss, valid_loss, accuracy
  warn(f'{self.__class__} conditioned on metric `{self.monitor}` which is not available. Available metrics are: {", ".join(map(str, self.learn.recorder.names[1:-1]))}')
```

3. To fix it you can tell it to monitor `valid_loss` by changing the callback to be:

``` python
learn = cnn_learner(data, models.resnet18, metrics=accuracy, callback_fns=partial(WandbCallback, monitor='valid_loss'))
```

I'm guessing that the name of the loss changed with fast.ai at some point.

### PR DESCRIPTION
**Pull request by borisdayma**
Make default "monitor" value consistent with update of fastai library -> changed from "val_loss" to "valid_loss".

fix #442

### CODE REVIEW

**Comment by borisdayma** [comment]
It seems like the change appeared at fastai version 1.0.53.
Since this framework evolves fast with breaking changes, instead of checking the version, we can default `monitor` to None and not initialize this value in the callback (which will then select default to whatever is the default at the time).

I tested with fastai version 1.0.52 & 1.0.55.
Warning message when using the wrong monitor value is given by fastai library.
